src/services/secrets-service.js

```diff
@@ -0,0 +1,105 @@
+import createDebug from 'debug';
+import fs from 'node:fs';
+import config, { sqlDir } from '../load-config.js';
+import { join } from 'node:path';
+import openDatabase from '../db.js';
+class SecretsDb {
+  constructor() {
+    this.debug = createDebug('actual:secrets-db');
+    this.db = null;
+    this.migrateNordigen();
+  }
+
+  /// Migrates nordigen from config.json or process.env to app secret
+  migrateNordigen() {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏instead of writing the migration in the codebase - could you use the `migrations`? Example here: https://github.com/actualbudget/actual-server/blob/master/migrations/1632571489012_remove_cache.js

Reply:
is it ok if i use the stuff from the secretService inside the nordigen_secret.js migration file then? or should i do it all "manually"

Reply:
Preferably manually. The shape & logic in this code will eventually evolve into something else. But the migration should stay exactly the same throughout its history.

Reply:
I cant figure out how to do the migration. Appreciate some help on this.

Reply:
Those are the migrations for the frontend database, I think we should remove them from this repo altogether. Since we don’t have migrations set up at the moment I don’t think it makes sense to set those up, we can just keep this chunk of code in here for ~6 months or so.

Reply:
Good point. I actually thought the migrations in `actual-server` were working.. but it turns out they don't 🤔 

So yes: lets keep your existing migration solution. Sorry for the noise.